src/nvim/api/private/helpers.c

```diff
@@ -770,7 +770,7 @@ void modify_keymap(Buffer buffer, bool is_unmap, String mode, String lhs,
   MapArguments parsed_args;
   memset(&parsed_args, 0, sizeof(parsed_args));
   if (parse_keymap_opts(opts, &parsed_args, err)) {
-    goto FAIL_AND_FREE;
+    goto fail_and_free;
```

Comment:
I like the caps, but the codebase never uses caps for labels so better to be consistent.